internal/command/arguments/test.go

```diff
@@ -60,6 +64,7 @@ func ParseTest(args []string) (*Test, tfdiags.Diagnostics) {
 	cmdFlags.StringVar(&test.JUnitXMLFile, "junit-xml", "", "junit-xml")
 	cmdFlags.BoolVar(&test.Verbose, "verbose", false, "verbose")
 	cmdFlags.IntVar(&test.OperationParallelism, "parallelism", DefaultParallelism, "parallelism")
+	cmdFlags.BoolVar(&test.Repair, "repair", false, "repair")
```

Comment:
I think this is technically also available to the main `test` command right? At least it will accept the flag and not error, can we work out a way so that `terraform test` will error if the repair flag is provided?